site/index.html

```diff
@@ -501,6 +501,37 @@ <h4 id="convolver">Convolver</h4>
                     <button id="play-drums" class="play action"></button>
                 </div>
 
+            <a name="tremolo"></a><h4>Tremolo</h4>
```

Comment:
OK, that's also in, and the demo is now playable. I'm going to see about retooling the effect for more options. I saw an alternate approach to a tremolo effect that uses a waveShaper node. Might look into that also. The public API should remain the same however.